index.js

```diff
@@ -1,34 +1,48 @@
 'use strict';
 
+const defer = require('p-defer');
+
+class AbortError extends Error {
+	constructor(message) { // eslint-disable-line no-useless-constructor
+		super(message);
```

Comment:
Done!

I'd briefly pushed a version that included `multipipe` as a dependency because I accidentally npm-installed that in the wrong terminal tab. Make sure you don't accidentally merge that one. Sorry about that.